packages/x-data-grid/src/constants/gridClasses.ts

```diff
@@ -578,6 +578,10 @@ export interface GridClasses {
    * Styles applied to the toggle of the grouping cell of the tree data.
    */
   treeDataGroupingCellToggle: string;
+  /**
+   * Styles applied to the loading container of the grouping cell of the tree data.
+   */
+  treeDataGroupingCellLoadingContainer: string;
```

Comment:
Is this something that we want to exposed to users? If not it might be nice to `@ignore` it.